terraform/ecs-service/iam.tf

```diff
@@ -201,3 +229,156 @@ data "aws_iam_policy_document" "execute_command_policy" {
     ]
   }
 }
+
+resource "aws_iam_policy" "custom_addons_policy" {
+  count       = var.iam_policy_addons_json != null ? 1 : 0
+  name        = "${local.service_name}-custom-addons-policy"
+  description = "Optional policy for custom permissions (addons) needed by the ECS task role"
+  policy      = var.iam_policy_addons_json
```

Comment:
struggling to find the AWS docs but we should be careful and add unit tests for the extreme number of the custom_addons_policy limits.
There is this post that indicates a 6k character limit, https://repost.aws/knowledge-center/iam-increase-policy-size.
I think at least a unit test would help confirm our size limit.

Reply:
I think that is good until someone hits it then we can address it